peertube/docker-compose.yaml

```diff
@@ -0,0 +1,77 @@
+services:
+  peertube:
+    # If you don't want to use the official image and build one from sources:
+    # build:
+    #   context: .
+    #   dockerfile: ./support/docker/production/Dockerfile.bookworm
+    image: chocobozzz/peertube:${PEERTUBE_IMAGE}
```

Comment:
I think PEERTUBE_IMAGE should be the whole image name + version.

Reply:
fixed in [a3a8e3a](https://github.com/EnigmaCurry/d.rymcg.tech/pull/260/commits/a3a8e3aff9db0591c542a3b2cd8f8b0a6c0a42f1)